### PR TITLE
Removed point/no-point, fixed retention-time - range-field exclusion

### LINKED ISSUE #83: Adjust manager to the change in the --point / --no-point approach
We decided to drop the possibility of not creating a mirror index. A mirror index is always created by default. Thus, there is no need for `--point` or `--no-point` flags. Since Manager has already been adjusted to these flags, it must re-adjusted again.

### PR DESCRIPTION
**Pull request by DawidNiezgodka**
Resolves #83 and #73.

### CODE REVIEW

**Review by raminqaf — CHANGES_REQUESTED** [review]
Please update the open API specs and generate them for the PR https://github.com/bakdata/quick-cli/pull/10

**Review by raminqaf — APPROVED** [review]
Looks good. Here the documentation how you can create the client code for the CLI
https://bakdata.github.io/quick/dev/developer/development/#openapi

File: manager/src/main/java/com/bakdata/quick/manager/mirror/resources/MirrorResourceLoader.java [hunk 1]
```diff
@@ -163,23 +162,21 @@ private Service createMirrorService(final String name) {
...
     private static Map<String, String> createCliArguments(final String topic,
         @Nullable final Duration retentionTime,
-        final boolean point,
         @Nullable final String rangeField) {
         final ImmutableMap.Builder<String, String> builder = ImmutableMap.<String, String>builder()
             .put("--input-topics", topic);
 
-        builder.put("--point", Boolean.toString(point));
+        if (Objects.nonNull(retentionTime) && Objects.nonNull(rangeField)) {
+            throw new BadArgumentException("The --range-field option must not be specified"
```
Comment (raminqaf) [inline manager/src/main/java/com/bakdata/quick/manager/mirror/resources/MirrorResourceLoader.java:170]: Extract the error message in a variable so it does not break in the line.

**Review by raminqaf — APPROVED** [review]

File: manager/src/main/java/com/bakdata/quick/manager/mirror/resources/MirrorResourceLoader.java (hunk 1 above)
Comment (DawidNiezgodka) [reply manager/src/main/java/com/bakdata/quick/manager/mirror/resources/MirrorResourceLoader.java]: Done. But is it really that relevant to not break it?

**Pull request merged**